design-system/packages/core/package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@babel/runtime": "^7.14.6",
     "@emotion/react": "^11.4.0",
+    "@types/facepaint": "1.2.1",
```

Comment:
This is because the `@types` packages make the things work globally so we didn't catch that the type is used in the generated declarations do this dep is needed